### ISSUE TITLE
About Ogbn-Arxiv.

### ISSUE THREAD

**Issue by Thetwotwo** [hengruizhang98/CCA-SSG#6]
I really like your work and try to do some outreach based on your work. When I perform experiments on the Ogbn-Arxiv dataset, I didn't get a great result. Have you run CCA-SSG on this dataset? If so, could you provide a configuration of the relevant hyperparameters? Thank you for your time and efforts.

**Comment by hengruizhang98** [comment]
Thanks for your interest. Did you add the reverse edges? I can achieve over 67% without fine-tuning the hyperparameters after adding the reverse edges. BTW, I  don't think ogbn-arxiv is a good dataset for evaluating SSL methods, as a randomly initialized GCN can achieve about 69% linear evaluation accuracy (with reverse edges).
